README.md

```diff
@@ -31,14 +31,14 @@ OpenShift engineers.
 **1 )** Exploratory data analysis of available data sources (gcsweb, TestGrid, Sippy) to understand the different data
 types that need to be addressed and identify what kind of ML approaches are possible.
 
-- Outcome will be notebooks that explain the data, how to access it and preprocess it in a data science friendly
+* Outcome will be notebooks that explain the data, how to access it and preprocess it in a data science friendly
   format (see this
-  [notebook for TestGrid data](https://github.com/aicoe-aiops/ocp-ci-analysis/blob/master/notebooks/TestGrid_EDA.ipynb)
+  [notebook for TestGrid data](notebooks/TestGrid_EDA.ipynb)
```

Comment:
@durandom this link causes the notebook file to be download on the [operate-first](https://www.operate-first.cloud/data-science/ocp-ci-analysis/) site is that the intended behavior or should it redirect to the notebook page on the site?